projects/core/src/lib/map/map-info/map-info.component.scss

```diff
@@ -0,0 +1,12 @@
+.map-info {
+    z-index: 1;
+    position: absolute;
+    margin-bottom: 10px;
```

Comment:
maybe better work with rem instead of pixels

Reply:
https://github.com/geo-engine/geoengine-ui/pull/720/commits/f9fa9764e7913450d7380b862ca6694e3092f31f